### PR TITLE
(ccs) new software release for auxtel and comcam clusters

### PR DESCRIPTION
**Pull request by glennmorris**

### CODE REVIEW

**Comment by jhoblitt** [comment]
It appears that the `()` section of this commit is truncated? [(site/tu/cluster) bump auxtel and comcam ccs ts_sal version](https://github.com/lsst-it/lsst-control/pull/873/commits/986d136a6d4af6b4b38a8050df51d8abe2c37159)

**Comment by glennmorris** [comment]
> It appears that the `()` section of this commit is truncated?

Sorry, do you mean the commit message?
I thought something like (site/tu/cluster/[comcam|auxtel]-ccs) would be hard to parse...

**Comment by jhoblitt** [comment]
> > It appears that the `()` section of this commit is truncated?
> 
> Sorry, do you mean the commit message? I thought something like (site/tu/cluster/[comcam|auxtel]-ccs) would be hard to parse...

Its fair game as long as it fits in 72 chars. If it is hard to summarize in a single line than breaking the change up into multiple commits is an option.

**Comment by glennmorris** [comment]
OK, I've reworded it, but I seem to be the only person using the {a|b} format.

**Review by tony-johnson — APPROVED** [review]
The changes look good to me.

**Review by jhoblitt — APPROVED** [review]
LGTM.
